tests/unit_tests/testing/test_sim_telarray_metadata.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/python3
+
+import copy
+import re
+from unittest.mock import MagicMock
+
+import numpy as np
+import pytest
+
+from simtools.simtel.simtel_io_metadata import read_sim_telarray_metadata
+from simtools.testing.sim_telarray_metadata import (
+    _sim_telarray_name_from_parameter_name,
+    assert_sim_telarray_metadata,
+    is_equal,
+)
+
+
+@pytest.fixture
+def test_metadata_file():
+    return (
+        "tests/resources/"
+        "run000010_gamma_za20deg_azm000deg_North_test_layout_6.0.0"
+        "_test-production-North.simtel.zst"
+    )
+
+
+@pytest.fixture
+def test_metadata(test_metadata_file):
+    return read_sim_telarray_metadata(test_metadata_file)
+
+
+def test_assert_sim_telarray_metadata_success(test_metadata_file):
+    """Test assert_sim_telarray_metadata with successful comparison."""
```

Comment:
This test is not for a success case, right?

Reply:
Correct. Fixed this.